### ISSUE TITLE
matching on document & sentence versions of corpora

### ISSUE THREAD

**Issue by hp0404** [hcss-utils/spacy-noun-phrases#9]
Why the results are different?

**Comment by hp0404** [comment]
- the sentence is: EU foreign policy chief Catherine Ashton urged Russia to withdraw troops from the border as a key step to de-escalate tensions.
- the document is (our sentence and the one preceeding it): Moscow has denied it is deliberately massing troops near Ukraine, saying the presence of its forces there is due to military exercises. EU foreign policy chief Catherine Ashton urged Russia to withdraw troops from the border as a key step to de-escalate tensions.
- the matcher should see that 'Ashton' has PERSON ENT_TYPE and is nsubj, 'urged' is a relevant verb, and Russia is obviously a relevant dobj; so it should match in both cases, but it only matches on the 'document' level, discarding match at the 'sentence' level
- so after inspecting both 'sentence' and 'document' versions with displacy, I noticed that spaCy parses the same sentence - EU foreign... - differently! 
  - when passed as a single sentence, it thinks that 'chief' is nsubj
  - when passed as a document, it thinks that 'Ashton' is nsubj

In short, the context matterns
